pull_request_template.md

```diff
@@ -0,0 +1,4 @@
+[] Wrote a test or tests for code you added.
+[] Linted the files you touched.
```

Comment:
Although this might not be able to work right out of the box, you should include Github actions do to first the linting, maybe you restrict it to a subset, and then subsets, and then finally all the code, until the tech debt is caught up with, that way this is not a manual checkbox list.